sdk/ai/azure-ai-inference/README.md

```diff
@@ -399,6 +399,7 @@ try:
     result = client.complete( ... )
 except HttpResponseError as e:
     print(f"Status code: {e.status_code} ({e.reason})")
+    print(f"{e.message}")
```

Comment:
nit: unnecessary f-string

Reply:
Good catch! per IM discussion, will fix this in the next release.